javascript/extractor/src/com/semmle/js/parser/TypeScriptParserMetadata.java

```diff
@@ -0,0 +1,67 @@
+package com.semmle.js.parser;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+
+/**
+ * Static data from the TypeScript compiler needed for decoding ASTs.
+ * <p>
```

Comment:
this `<p>` doesn't have a closing  `</p>`.

Reply:
It's not needed and our convention is to omit `</p>`. And indeed our autoformatter will remove it.